src/main/java/edu/harvard/hul/ois/fits/consolidation/OISConsolidator.java

```diff
@@ -91,6 +91,10 @@ public OISConsolidator(Fits fits) throws FitsConfigurationException {
 		this.fits = fits;
 		reportConflicts = fits.getConfig().getBoolean("output.report-conflicts",true);
 		displayToolOutput = fits.getConfig().getBoolean("output.display-tool-output",false);
+		// tool output can be turned on from command line overriding configured value
+		if (fits.isRawToolOutput()) {
```

Comment:
It might be simpler just to write:

```java
displayToolOutput = fits.isRawToolOutput();
```

Reply:
My thought was to keep the value in fits.xml if it happened to be 'true'.  It looks like with this proposed change, that if there's no command line option, and the fits.xml value is 'true', then there would be no tool output.

Reply:
I gotcha now. I misunderstood how the two different options were being resolved.